setup.py

```diff
@@ -7,7 +7,7 @@
 
 NAME = "Orange3-Explain"
 
-VERSION = "0.5.3"
+VERSION = "0.5.4"
```

Comment:
Version should be changed in the separate release commit when it is performed. 

```suggestion
VERSION = "0.5.3"
```

Reply:
Yes, but I changed it in order to enable automatic install when drag and doped the .zip file into the add-on dialog (without making a release).
I think it should be ok, if we change it to 0.6.0 at the next release and just ignore the missing tag 0.5.4.

Reply:
@VesnaT, so we have a bug in the installation system? I think, ideally, the drag should always reinstall regardless of the version.